addons/membership/test/test_membership.yml

```diff
@@ -1,3 +1,17 @@
+-
```

Comment:
The idea of this task was to fix the module, assuming that the tests we correct... can you explain why you added changes in tests (and new tests)? It is probably a good idea to extend existing tests, but can you move them in a 2nd commit, so that we can easily check that the module is fixed, even without the new tests. Thanks!